app/src/main/java/net/aiscope/gdd_app/ui/capture/CaptureImageActivity.kt

```diff
@@ -54,6 +55,23 @@ class CaptureImageActivity : AppCompatActivity(), CaptureImageView, CaptureFlow
                 cameraErrorCallback = { presenter.onCaptureError(it) }
             )
             zoomController = ZoomController(fotoapparat, cameraZoomLevel, cameraView)
+            toast  = Toast.makeText(
+                this@CaptureImageActivity,
+                R.string.you_cannot_take_a_picture_while_zooming,
+                Toast.LENGTH_SHORT);
+
+            zoomController.onZoomChangedListener = ZoomController.OnZoomChangedListener {
+                if (it)
+                {
+                    captureImageButton.isEnabled = false;
+                    toast.show()
+                }
+                else
+                {
+                    captureImageButton.isEnabled = true;
```

Comment:
I think that using an explicit parameter would make this more readable, do you agree? Also, please stick to the project conventions of where to place the curly braces, and no semicolons, for consistency reasons.

```suggestion
                Toast.LENGTH_SHORT)

            zoomController.onZoomChangedListener = ZoomController.OnZoomChangedListener { isZoomed ->
                if (isZoomed) {
                    captureImageButton.isEnabled = false
                    toast.show()
                } else {
                    captureImageButton.isEnabled = true
```